CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ else()
 endif()
 
 # Thrust project
-project(thrust VERSION 2.5.0.0 LANGUAGES CXX)
```

Comment:
We have this change in this pull request. Please approve that first:
https://github.com/ROCmSoftwarePlatform/rocThrust/pull/39